### ISSUE TITLE
RTD builds are failing

### ISSUE THREAD

**Issue by emlys** [natcap/invest.users-guide#62]
Example: https://readthedocs.org/projects/invest-userguide/builds/15753127/
The error is 
```
...
File "/home/docs/checkouts/readthedocs.org/user_builds/invest-userguide/conda/latest/lib/python3.8/site-packages/natcap/invest/annual_water_yield.py", line 10, in <module>
    import pygeoprocessing
  File "/home/docs/checkouts/readthedocs.org/user_builds/invest-userguide/conda/latest/lib/python3.8/site-packages/pygeoprocessing/__init__.py", line 9, in <module>
    from . import geoprocessing
  File "/home/docs/checkouts/readthedocs.org/user_builds/invest-userguide/conda/latest/lib/python3.8/site-packages/pygeoprocessing/geoprocessing.py", line 16, in <module>
    from . import geoprocessing_core
  File "src/pygeoprocessing/geoprocessing_core.pyx", line 1, in init pygeoprocessing.geoprocessing_core
ValueError: numpy.ndarray size changed, may indicate binary incompatibility. Expected 96 from C header, got 88 from PyObject

Exception occurred:
  File "src/pygeoprocessing/geoprocessing_core.pyx", line 1, in init pygeoprocessing.geoprocessing_core
ValueError: numpy.ndarray size changed, may indicate binary incompatibility. Expected 96 from C header, got 88 from PyObject
The full traceback has been saved in /tmp/sphinx-err-75_1md82.log, if you want to report the issue to the developers.
Please also report this if it was a user error, so that a better error message can be provided next time.
A bug report can be filed in the tracker at <https://github.com/sphinx-doc/sphinx/issues>. Thanks!
```

I was getting the same error message in invest, but for scenic quality not pygeoprocessing, before fixing natcap/invest#815. The build on `latest` last passed on Jan 4 and first failed with this error on Jan 5.

**Comment by phargogh** [comment]
This is almost always an issue arising when:

1. InVEST has been installed from a wheel and
2. The numpy version installed is older than the version of numpy that InVEST was built against.

Compiling `natcap.invest` from source is probably the safest way around the error in the short-term, but I'm also wondering if we need to do some extra configuration in `pyproject.toml` to try to build InVEST against more specific (earlier) versions of `numpy` to help avoid this issue.

**Comment by emlys** [comment]
This was fixed in the 3.10.2 release.
